lib/conjur/conjur_config.rb

```diff
@@ -23,7 +23,8 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
-      authenticators: []
+      authenticators: [],
+      tracing_enabled: false
```

Comment:
You could probably piggyback off of the existing [conjur_config_spec](https://github.com/cyberark/conjur/blob/master/spec/lib/conjur/conjur_config_spec.rb) to add a couple examples verifying that your additional property is indeed loading as expected from the config file and/or env vars.

It may also warrant an additional test to verify that the trace method is getting called for a given controller action